PyInstaller/hooks/hook-sphinx.py

```diff
@@ -73,8 +73,12 @@
 # Sphinx 1.3.1 adds additional mandatory dependencies unconditionally imported
 # by the "sphinx.themes" module regardless of the current Sphinx configuration:
 # the "alabaster" and "sphinx_rtd_theme" themes, each relying on data files.
-if is_module_satisfies('sphinx >= 1.3.1') and is_module_satisfies('sphinx < 1.4'):
+if is_module_satisfies('sphinx >= 1.3.1') and is_module_satisfies(
+        'sphinx < 1.4'):
     datas.extend(collect_data_files('alabaster'))
     datas.extend(collect_data_files('sphinx_rtd_theme'))
 elif is_module_satisfies('sphinx >= 1.3.1'):
     datas.extend(collect_data_files('alabaster'))
+
+if is_module_satisfies('sphinx >= 1.6.1'):
+    hiddenimports += collect_submodules('sphinx.environment.collectors')
```

Comment:
👍 That's been quick :-) But what should the commit massage "yapf" mean? Please use something less cryptic :-)